src/js/view/controls/controlbar.js

```diff
@@ -270,7 +275,12 @@ export default class Controlbar {
 
         // Listen for model changes
         _model.change('volume', this.onVolume, this);
-        _model.change('mute', this.onMute, this);
+        _model.change('mute', (model, muted) => {
+            this.renderVolume(muted, model.get('volume'));
+            const muteText = muted ? localization.unmute : localization.mute;
+            muteTip.setText(muteText);
+            setAttribute(this.elements.volumetooltip.element(), 'aria-label', muteText);
```

Comment:
You can just use the `volumeTooltopEl` variable you created on L226

```suggestion
             setAttribute(volumeTooltipEl, 'aria-label', muteText); 
```